pkgs/development/libraries/libantlr3cpp/default.nix

```diff
@@ -0,0 +1,25 @@
+{stdenv, fetchgit}:
+
+stdenv.mkDerivation {
+  name = "libantlr3cpp-3.5";
+  src = fetchgit {
+    url = "https://github.com/antlr/antlr3.git";
+    rev = "5c2a916a10139cdb5c7c8851ee592ed9c3b3d4ff";
+    sha256 = "1i0w2v9prrmczlwkfijfp4zfqfgrss90a7yk2hg3y0gkg2s4abbk";
+    fetchSubmodules = false;
+  };
+
+  dontBuild = true;
```

Comment:
What is the context for not triggering the build phase?

Reply:
Because these are header files only - I'm simply copying them from github. Now that you mention it, probably no need for a `stdenv.mkDerivation` then?

Reply:
Oh sorry! I accidentally pressed the resolve conversation button, because I have fat fingers.

We can build libraries into shared object and development headers separately, namely `out` and `dev` outputs.

We can ship this first and I can show how this is done

Reply:
Awesome! Please show me how :)